tableland-openapi-spec.yaml

```diff
@@ -286,6 +320,57 @@ paths:
                   - message
                 example:
                   message: "Invalid id format"
+  /schema/{tableName}:
+    get:
+      summary: Get the schema of a table by its full name
+      parameters:
+        - in: path
+          name: tableName
+          schema:
+            type: string
+          required: true
+          description: The full table of the name
```

Comment:
Nit: maybe wanted to write "Full name of the table", or "Full table name" :smile: